### PR TITLE
feat: request node 0.29.0 + tag defaults to appVersion

### PR DESCRIPTION
**Pull request by benjlevesque**
This release is a breaking change and requires configuration update

Included in this PR:
- use the Chart.AppVersion as the default tag
- introduce a new GH action to patch the AppVersion, for easy update

### CODE REVIEW

File: request-node/values.yaml [hunk 1]
```diff
@@ -35,8 +35,8 @@ nodeIndexDirectoryPath: /index
 # The Request Node image information
 nodeImage:
   image: requestnetwork/request-node
-  tag: 0.8.0
-  pullPolicy: Always
+  tag: 0.27.0
```
Comment (benjlevesque) [inline request-node/values.yaml:38]:
```suggestion
  tag: 0.29.0
```

File: request-node/values.yaml [hunk 2]
```diff
@@ -35,8 +35,8 @@ nodeIndexDirectoryPath: /index
 # The Request Node image information
 nodeImage:
   image: requestnetwork/request-node
-  tag: 0.8.0
-  pullPolicy: Always
+  tag: 0.29.0
+  pullPolicy: IfNotPresent
```
Comment (benjlevesque) [inline request-node/values.yaml:39]: just cleaning up defaults

File: request-node/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 0.8.0
```
Comment (alexandre-abrioux) [inline request-node/Chart.yaml:2]:
```suggestion
appVersion: 0.29.0
```

File: request-node/Chart.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
...
 description: A Helm chart for Request Node
 name: request-node
-version: 0.6.18
+version: 0.6.19
```
Comment (alexandre-abrioux) [inline request-node/Chart.yaml:5]: Shouldn't we increase the major version?

**Review by alexandre-abrioux — APPROVED** [review]

File: request-node/templates/statefulSet.yaml [hunk 5]
```diff
@@ -73,11 +73,9 @@ spec:
             value: {{ required "nodeEnv.networkId is required" .Values.nodeEnv.networkId | quote }}
           - name: LOG_LEVEL
             value: {{ .Values.nodeEnv.logLevel }}
-          - name: IPFS_PORT
-            value: {{ .Values.ipfs.api.port | quote  }}
           {{ if not .Values.ipfs.sidecar }}
-          - name: IPFS_HOST
-            value: {{ required "ipfs.api.host is required when IPFS is not a sidecar" .Values.ipfs.api.host | quote }}
+          - name: IPFS_URL
+            value: {{ required "ipfs.api.url is required when IPFS is not a sidecar" .Values.ipfs.api.url | quote }}
```
Comment (benjlevesque) [inline request-node/templates/statefulSet.yaml:78]: new config

File: request-node/templates/statefulSet.yaml [hunk 6]
```diff
@@ -25,7 +25,7 @@ spec:
       containers:
       # Request Node Container
       - name: {{ .Chart.Name }}
-        image: "{{ .Values.nodeImage.image }}:{{ .Values.nodeImage.tag }}"
+        image: "{{ .Values.nodeImage.image }}:{{ .Values.nodeImage.tag | default .Chart.AppVersion }}"
```
Comment (benjlevesque) [inline request-node/templates/statefulSet.yaml:28]: default to AppVersion

**Comment by benjlevesque** [comment]
@alexandre-abrioux can you take a look at the new changes?

**Review by alexandre-abrioux — APPROVED** [review]

**Comment by MantisClone** [comment]
LGTM :+1:

**Pull request merged**